src/ui/Streamline/Streamline.tsx

```diff
@@ -0,0 +1,26 @@
+import { classNames } from './classnames';
```

Comment:
```suggestion
import { classNames } from './classNames';
```

тут с импортом промахнулся